### PR TITLE
Update 0.129 release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.129.rst [hunk 1]
```diff
@@ -2,26 +2,34 @@
...
-
 General Changes
 ---------------
 
+* Fix a planner issue that could cause queries involving ``OUTER JOIN`` to
+  return incorrect results.
+* Some queries, particularly those using :func:`max_by` or :func:`min_by`, now
+  accurately reflect their true memory usage and thus appear to use more memory
+  than before.
+* Always clear screen when canceling query with ``ctrl-C``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: Why did you remove the "CLI Changes" section?

File: presto-docs/src/main/sphinx/release/release-0.129.rst [hunk 2]
```diff
@@ -33,4 +41,4 @@ Hive Changes
   data outside the current region.
 * Server-side encryption is now supported for S3. To enable this feature,
   use ``hive.s3.sse.enabled=true`` in your Hive catalog properties file.
-* Add hive table property ``retention_days``.
+* Add Hive table property ``retention_days``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: The "Hive" here is redundant since this is the "Hive changes" section

File: presto-docs/src/main/sphinx/release/release-0.129.rst [hunk 3]
```diff
@@ -2,26 +2,34 @@
...
+* Fix hang in large queries with ``ORDER BY`` and ``LIMIT``.
+* Fix an issue when casting empty arrays or arrays containing only ``NULL`` to
+  other types.
+* Table property names are now properly treated as case-insensitive.
 * The JDBC driver now properly supports non-query statements.
   The ``Statement`` interface supports all variants of the ``execute`` methods.
   It also supports the ``getUpdateCount`` and ``getLargeUpdateCount`` methods.
-* Fix :doc:`/sql/show-session` to not show hidden session properties.
-* Fix hang in large queries with ``ORDER BY`` and ``LIMIT``.
+* Improve performance of ``INSERT`` or ``CREATE TABLE`` queries involving ``UNION ALL``.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: This should be "CREATE TABLE AS". How about
```
* Improve performance of queries involving ``UNION ALL`` that write data.

File: presto-docs/src/main/sphinx/release/release-0.129.rst [hunk 4]
```diff
@@ -2,26 +2,34 @@
...
+* Introduce ``P4HyperLogLog`` type, which uses an implementation of the HyperLogLog data
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: "Introduce the"

File: presto-docs/src/main/sphinx/release/release-0.129.rst (hunk 1 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: There was only one entry, so it didn't seem worth having an entire section for it.

File: presto-docs/src/main/sphinx/release/release-0.129.rst (hunk 2 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.129.rst]: Should we remove Hive from "Hive catalog properties" in the line above, then?

**Comment by electrum** [comment]
Additions:
```
General:
* Throw exception when using :doc:`/sql/set-session` or :doc:`/sql/reset-session`
  from JDBC rather than silently ignoring the command.
* Minor UI improvements for query detail page.
* Do not display useless stack traces for expected exceptions in verifier.
CLI:
* Make client request timeout configurable.
Hive:
* Add support for S3 ``EncryptionMaterialsProvider``.
